File: 5630start.cpp
#include <iostream>
#include <cstdlib> // For std::atoi and std::atof

int main(int argc, char* argv[]) {
    // Check if the user provided the correct number of arguments
    if (argc != 6) {
        std::cerr << "Usage: " << argv[0] << " <int1> <int2> <double> <double> <double>\n";
        return 1; // Exit with error code
    }

    // Parse the command-line arguments
    int d = std::atoi(argv[1]);
    int N = std::atoi(argv[2]);
    double r1 = std::atof(argv[3]);
    double r2 = std::atof(argv[4]);
    double a = std::atof(argv[5]);
    
    // ******* add your code here 

    double volume = 0;
    double stdev = 0;

    // *******

    // do not change the format below
    std::cout << "(r1,r2): " << r1 << " " << r2 << std::endl;
    std::cout << "(d,N,a): " << d << " " << N << " " << a << std::endl;
    std::cout << "volume: " << volume << std::endl;
    std::cout << "stat uncertainty: " << stdev << std::endl;

    return 0; // Exit successfully
}

File: 5630start.py
import sys

def main():
    # Check if the user provided the correct number of arguments
    if len(sys.argv) != 6:
        print(f"Usage: {sys.argv[0]} <int1> <int2> <double> <double> <double>")
        sys.exit(1)  # Exit with error code

    # Parse the command-line arguments
    try:
        d = int(sys.argv[1])         # First integer
        N = int(sys.argv[2])         # Second integer
        r1 = float(sys.argv[3])      # First double
        r2 = float(sys.argv[4])      # Second double
        a = float(sys.argv[5])       # Third double
    except ValueError:
        print("Error: Please provide valid integers and doubles as arguments.")
        sys.exit(1)  # Exit with error code

    # ******* Add your code here

    volume = 0.0
    stdev = 0.0

    # *******

    # Do not change the format below
    print(f"(r1,r2): {r1} {r2}")
    print(f"(d,N,a): {d} {N} {a}")
    print(f"volume: {volume}")
    print(f"stat uncertainty: {stdev}")

if __name__ == "__main__":
    main()
